### PR TITLE
DAOS-9906 doc: fix copyright style

### PR DESCRIPTION
**Pull request by knard-intel**
Fix files with inconsistent copyright style disabling git hooks.

Signed-off-by: Cedric Koch-Hofer <cedric.koch-hofer@intel.com>

### CODE REVIEW

File: ftest.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (C) Copyright 2019-2020 Intel Corporation
+# Copyright 2019-2020 Intel Corporation
```
Comment (ashleypittman) [inline ftest.sh:2]: The commit-hook clearly isn't working for this one.

Comment (knard-intel) [reply ftest.sh:2]: not sure to understand the implication of this remark: Do I have to keep as it was before or my fix is correct ?

Comment (ashleypittman) [reply ftest.sh:2]: Are you using the commit-hook?  Whilst the format appears correct the date does not include 2022, so either you're not using the hook or it's still not picking up this file for some reason.

Having looked closer the same comment applies to almost all files in this PR.

Comment (knard-intel) [reply ftest.sh:2]: From the description of the ticket https://daosio.atlassian.net/browse/DAOS-9906, I understood that all the Copyright shall be consistent and respect the following format:
Copyright 20...
However, I could have misunderstood the ticket description. From what I understood it was not limited to Copyright containing 2022.

Comment (knard-intel) [reply ftest.sh:2]: I will start looking why the git hooks are not working. Until, I have find a fix to this I will close this ticket and I will reopen it when I will have a proposal to fix it.

**Pull request closed without merging**